setup.py

```diff
@@ -33,7 +30,7 @@
     maintainer_email='dev-platform@dropbox.com',
     url='https://github.com/dropbox/stone',
     install_requires=install_reqs,
-    license=LICENSE,
+    license=open('LICENSE').read(),
```

Comment:
Technically, this should probably be a string like "MIT" (if that's what our license is). If it is a known license, we should probably add a corresponding `License :: …` [classifier](https://pypi.python.org/pypi?%3Aaction=list_classifiers).